bcda/api/v1/api_test.go

```diff
@@ -821,71 +821,71 @@ func (s *APITestSuite) TestHealthCheck() {
 	assert.Equal(s.T(), http.StatusOK, s.rr.Code)
 }
 
-func (s *APITestSuite) TestAuthInfoDefault() {
-
-	// get original provider so we can reset at the end of the test
-	originalProvider := auth.GetProviderName()
-
-	// set provider to bogus value and make sure default (alpha) is retrieved
-	auth.SetProvider("bogus")
-	req := httptest.NewRequest("GET", "/_auth", nil)
-	handler := http.HandlerFunc(GetAuthInfo)
-	handler.ServeHTTP(s.rr, req)
-	assert.Equal(s.T(), http.StatusOK, s.rr.Code)
-	respMap := make(map[string]string)
-	err := json.Unmarshal(s.rr.Body.Bytes(), &respMap)
-	if err != nil {
-		s.T().Error(err.Error())
-	}
-	assert.Equal(s.T(), "alpha", respMap["auth_provider"])
-
-	// set provider back to original value
-	auth.SetProvider(originalProvider)
-}
-
-func (s *APITestSuite) TestAuthInfoAlpha() {
-
-	// get original provider so we can reset at the end of the test
-	originalProvider := auth.GetProviderName()
-
-	// set provider to alpha and make sure alpha is retrieved
-	auth.SetProvider("alpha")
-	req := httptest.NewRequest("GET", "/_auth", nil)
-	handler := http.HandlerFunc(GetAuthInfo)
-	handler.ServeHTTP(s.rr, req)
-	assert.Equal(s.T(), http.StatusOK, s.rr.Code)
-	respMap := make(map[string]string)
-	err := json.Unmarshal(s.rr.Body.Bytes(), &respMap)
-	if err != nil {
-		s.T().Error(err.Error())
-	}
-	assert.Equal(s.T(), "alpha", respMap["auth_provider"])
-
-	// set provider back to original value
-	auth.SetProvider(originalProvider)
-}
-
-func (s *APITestSuite) TestAuthInfoOkta() {
-
-	// get original provider so we can reset at the end of the test
-	originalProvider := auth.GetProviderName()
-
-	// set provider to okta and make sure okta is retrieved
-	auth.SetProvider("okta")
-	req := httptest.NewRequest("GET", "/_auth", nil)
-	handler := http.HandlerFunc(GetAuthInfo)
-	handler.ServeHTTP(s.rr, req)
-	assert.Equal(s.T(), http.StatusOK, s.rr.Code)
-	respMap := make(map[string]string)
-	err := json.Unmarshal(s.rr.Body.Bytes(), &respMap)
-	if err != nil {
-		s.T().Error(err.Error())
-	}
-	assert.Equal(s.T(), "okta", respMap["auth_provider"])
-
-	// set provider back to original value
-	auth.SetProvider(originalProvider)
-}
+// func (s *APITestSuite) TestAuthInfoDefault() {
```

Comment:
Test that may not apply.

Some should be tested by SSAS, and the rest need revamping.